### DISCUSSION TITLE
[@types/tabulator-tables] missing/wrong items

### DISCUSSION THREAD

**Discussion by Martino-Martinez** [DefinitelyTyped/DefinitelyTyped#68711] [Issues with a @types package]
Interface ColumnDefinition
- maxInitialWidth is missing
- See https://tabulator.info/docs/5.6/columns#definition

Interface OptionRows
- All selectable... options have been renamed to selectableRows...
- See https://tabulator.info/docs/5.6/upgrade
- Since there is now a range selection module, all selectable... options should be moved to a separate interface (for example OptionsRangeSelection) and this should be included in interface Options

I do not have enough experience to create a pull request with these changes. Maybe someone with more experience could do this.

Many thanks!
-- Martin

[a comment by an automated account is not shown]

**Comment by lschierer** [comment]
mutateLink also appears to be missing from the column definition type.
